lib/matplotlib/tests/test_figure.py

```diff
@@ -287,6 +287,16 @@ def test_suptitle_fontproperties():
     assert txt.get_weight() == fps.get_weight()
 
 
+@image_comparison(['figure_suptitle_subfigures'],
+                  extensions=['png'], style='mpl20')
```

Comment:
I'm not sure we really need a visual test here...  It should be enough to check that facecolor="none" for the subfigure by default, no?

Reply:
I can see either way. I guess removing the visual test would keep the build time minimal. However, having the visual test provides some protection if another subtle change impacts the way the subfigures can hide the suptitle. If you think that risk is minimal because the behavior is well understood, I'm happy to remove it.

Reply:
The suptitle is under the subfigure (if subfigure is called first).  There isn't much to be done about that without using a layout manager.  So I'd argue either the subfigure has a clear face or it doesn't is all we can test here.  If something else ends up in the subfigure that is not clear, then the suptitle will disappear anyway.

Reply:
Ok I will update the PR to:
(1) Remove the visual test
(2) Add a new test that creates the figure and checks for the default clear face on the subfigures. (And maybe a non-default value test too.)